aboveFold.css

```
@@ -406,9 +424,8 @@
         padding-right: 40px;
         margin-left: auto;
```

All margins can be written as one line
```suggestion
        margin: 0 auto;
```